### DISCUSSION TITLE
Custom Inflector pluralizations.

### DISCUSSION THREAD

**Discussion by cristian-home** [laravel/framework#33133] [General]
- Laravel Version: 7.14.1
- PHP Version: 7.4.3
- Database Driver & Version: SQLite

### Description:
In my project (in spanish) in Laravel 6 I created a service provider to add custom pluralizations to my application, for them I created a rule set for Inflector in the `Register()` method.

```
public function register()
  {
    /**
     * Extender pluralización para palabras irregulares
     */
    Inflector::rules('plural', [
      'irregular' => [
        "organizacion" => "organizaciones",
        "proovedor" => "proovedores",
        "ubicacion" => "ubicaciones",
        "estacion" => "estaciones",
        "operador" => "operadores",
        "sucursal" => "sucursales",
        "entidad" => "entidades",
        "ciudad" => "ciudades",
        "legal" => "legales",
        "pais" => "paises"
      ]
    ]);
  }
```
This helped me in creating a model with its respective controller, migration and factory, for example: `php artisan make:model Organizacion -a` creates table names correctly as "organizaciones" and also helpme with route model binding.

In laravel 7 it seems that inflector uses a new api, initially it alerted me with that the class `Doctrine\Common\Inflector` didn't exist, I installed the version 1.4 of inflector, but Laravel doesn't take my pluralizations anymore and I don't know how to accomplish this in Laravel 7.

Please help me.

**Comment by driesvints** [comment]
Moving to discussions.

**Reply by cristian-home** [reply]
Why?
